odbc-api/tests/integration.rs

```diff
@@ -2670,9 +2670,15 @@ fn columns_query(profile: &Profile) {
             .unwrap()
             .into_iter(),
     );
-    let columns = conn
-        .columns(&conn.current_catalog().unwrap(), "dbo", table_name, "a")
-        .unwrap();
+    // Mariadb does not support schemas
```

Comment:
May I suggest rather than matching on the profile to parameterize the schema, based on it. E.g.:

```rust
#[test_case(MSSQL, dbo; "Microsoft SQL Server")]
#[test_case(MARIADB, ""; "Maria DB")]
#[test_case(SQLITE_3, ""; "SQLite 3")]
fn columns_query(profile: &Profile, schema: &str) {
    //...
}
```

Reply:
Done